### ISSUE TITLE
TabbedPage Indicates Multiple Item Selection

### ISSUE THREAD

**Issue by greatoceansoftware** [dotnet/maui#6751]
### Description

This project demonstrates a bug in TabbedPage showing multiple selection.

The reproduction project is [here](https://github.com/greatoceansoftware/TabbedPageMultipleSelectedItems).

EDIT: Actually, this was built/discovered on RC 2.

![2022-05-02_08-30-59](https://user-images.githubusercontent.com/78753788/166265291-15e2b7cd-0ad2-41e2-a3bc-1864eaf33769.png)

### Steps to Reproduce

1. Build and run the project as Windows Machine.
2. Click the Project button. The Project TabbedPage appears.
3. Click the Activities tab.
4. Click the Flyout (hamburger menu button), and choose Archive.
5. Click the Archive button. 
6. A new Project TabbedPage appears with the Stakeholders tab and the Risks tab selected, which should not be possible.

### Version with bug

Release Candidate 1

### Last version that worked well

Unknown/Other

### Affected platforms

Windows, I was *not* able test on other platforms

### Affected platform versions

Windows 10.0.19044 Build 19044

### Did you find any workaround?

No

**Comment by kristinx0211** [comment]
verified repro on windows with above project.

**Comment by kristinx0211** [comment]
still an issue on windows with 17.3.0 Preview 2.0 [32509.345.main].

**Comment by greatoceansoftware** [comment]
Thanks. Unfortunately, I've decided to port my WPF application to WinUI3 instead of MAUI, establishing a Windows version, then using UNO to target other desktop platforms, preferably via WASM. I did not want to invest further time in the platform, along with Blazor, which would be totally new to me. The route I've chosen is very familiar and the tools seem better documented and stable, to this point.

I think MAUI is better suited to those coming from Xamarin and targeting mobile. WinUI3 is much better for those of us coming from WPF. Sorry I couldn't be of further assistance.
